statistics-dashboard-common/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>statistics-dashboard-backend</artifactId>
+    <groupId>st.statistics.backend</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>statistics-dashboard-common</artifactId>
+  <packaging>jar</packaging>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
```

Comment:
You are using this snippet everywhere... no need to declare java version separately. Perhaps there is a way to put this in the parent pom only? Alternatively, take a look at how we declare this in Clio (using the `<build>` construct).

Reply:
These were put automatically when the project was created. I will have a look into clio to fix this then